generate.py

```diff
@@ -0,0 +1,305 @@
+# Copyright 2023 The US Pilot Synthetic Data Gen Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script to generate impressions for a single Campaign."""
+
+
+from dataclasses import dataclass, asdict
+import datetime
+import numpy as np
+from scipy import stats
+import operator
+import inspect
+import csv
+import math
+from functools import reduce
+import itertools
+
+
+def get_virtual_people_buckets(bucket_probs):
+    ranges = []
+    start = 0
+    for bucket, prob in bucket_probs:
+        width = int(prob * NUM_VIRTUAL_PEOPLE)
+        ranges.append((bucket, range(start, start + width)))
+        start += width
+    return ranges
+
+
+# Total number of Virtual people, per the US population
+NUM_VIRTUAL_PEOPLE = 60_000_000
```

Comment:
Since it's in the "US" repo, would you change this to the US population? (are we gonna do load test with this synthetic data?)